### DISCUSSION TITLE
configuration/support for CYD (cheap yellow display) available?

### DISCUSSION THREAD

**Discussion by chriskuku** [peterhinch/micropython-nano-gui#63] [General]
Hi. Thanks for making this GUI available. Just a question: is there a configuration for the CYD or CYD2USB available?

**Comment by peterhinch** [comment]
I can't find a definitive datasheet for the CYD but a quick websearch indicates that it uses an ILI9341 display controller. If this is correct, there is a [suitable driver](https://github.com/peterhinch/micropython-nano-gui/blob/master/DRIVERS.md#32-drivers-for-ili9341).

**Comment by peterhinch** [comment]
I found [this reference](https://github.com/witnessmenow/ESP32-Cheap-Yellow-Display/blob/main/PINS.md) which should enable you to get this working. Note that the touch controller is an XPT2046 which is supported by [micropython-touch](https://github.com/peterhinch/micropython-touch) so it shoud work with that GUI also.

If you produce a working config file for either of these GUI's please let me know and I will post it as an example (crediting you, of course).

**Reply by chriskuku** [reply]
Thanks a lot!

**Reply by Sposch123** [reply]
I am looking at the Same topic. Have you been able to use micro GUI with the CYD?  Thanks!

**Reply by chriskuku** [reply]
Sorry, I discontinued my own project due to other work that came in the way.
